Build failed: Ambiguous occurrence ‘take’

```
$ cabal install webify
Resolving dependencies...
Configuring binary-strict-0.4.8.3...
Configuring blaze-builder-0.4.0.2...
Configuring hopfli-0.2.2.1...
Configuring colour-2.3.4...
Building binary-strict-0.4.8.3...
Building blaze-builder-0.4.0.2...
Building hopfli-0.2.2.1...
Building colour-2.3.4...
Installed blaze-builder-0.4.0.2
Configuring primitive-0.6.3.0...
Building primitive-0.6.3.0...
Installed colour-2.3.4
Downloading transformers-compat-0.5.1.4...
Installed binary-strict-0.4.8.3
Downloading xmlgen-0.6.2.1...
Installed hopfli-0.2.2.1
Configuring xmlgen-0.6.2.1...
Downloading ansi-terminal-0.8.0.1...
Configuring ansi-terminal-0.8.0.1...
Building xmlgen-0.6.2.1...
Building ansi-terminal-0.8.0.1...
Installed primitive-0.6.3.0
Downloading vector-0.12.0.1...
Configuring transformers-compat-0.5.1.4...
Building transformers-compat-0.5.1.4...
Configuring vector-0.12.0.1...
Installed transformers-compat-0.5.1.4
Building vector-0.12.0.1...
Installed ansi-terminal-0.8.0.1
Downloading ansi-wl-pprint-0.6.8.2...
Configuring ansi-wl-pprint-0.6.8.2...
Installed xmlgen-0.6.2.1
Building ansi-wl-pprint-0.6.8.2...
Installed ansi-wl-pprint-0.6.8.2
Downloading optparse-applicative-0.14.0.0...
Configuring optparse-applicative-0.14.0.0...
Building optparse-applicative-0.14.0.0...
Installed optparse-applicative-0.14.0.0
Installed vector-0.12.0.1
Downloading webify-0.1.8.0...
Configuring webify-0.1.8.0...
Building webify-0.1.8.0...
Failed to install webify-0.1.8.0
Build log ( /home/desrum_m/.cabal/logs/ghc-8.2.2/webify-0.1.8.0-7GMpEY8THTv1aWpncJGm9y.log ):
cabal: Entering directory '/tmp/cabal-tmp-11363/webify-0.1.8.0'
Configuring webify-0.1.8.0...
Preprocessing executable 'webify' for webify-0.1.8.0..
Building executable 'webify' for webify-0.1.8.0..
[1 of 8] Compiling Utils            ( src/Utils.hs, dist/build/webify/webify-tmp/Utils.dyn_o )
[2 of 8] Compiling Font             ( src/Font.hs, dist/build/webify/webify-tmp/Font.dyn_o )
[3 of 8] Compiling TTF              ( src/TTF.hs, dist/build/webify/webify-tmp/TTF.dyn_o )

src/TTF.hs:471:21: error:
    Ambiguous occurrence ‘take’
    It could refer to either ‘Data.List.take’,
                             imported from ‘Data.List’ at src/TTF.hs:30:1-26
                             (and originally defined in ‘GHC.List’)
                          or ‘Data.Map.take’,
                             imported from ‘Data.Map’ at src/TTF.hs:31:1-63
                             (and originally defined in ‘Data.Map.Internal’)
    |
471 |   zipWith getGlyph (take glyphCount offsets) $ diff offsets
    |                     ^^^^
cabal: Leaving directory '/tmp/cabal-tmp-11363/webify-0.1.8.0'
cabal: Error: some packages failed to install:
webify-0.1.8.0-7GMpEY8THTv1aWpncJGm9y failed during the building phase. The
exception was:
ExitFailure 1
```

--- a/src/TTF.hs
+++ b/src/TTF.hs
@@ -468,7 +468,7 @@ parseGlyf numberOfContours | numberOfContours >= 0 = do
 
 parseGlyfs :: Int -> [Int] -> Map String TableDirectory -> B.ByteString -> [Glyf]
 parseGlyfs glyphCount offsets tds font =
-  zipWith getGlyph (take glyphCount offsets) $ diff offsets
+  zipWith getGlyph (Data.List.take glyphCount offsets) $ diff offsets
   where tableStart = fromIntegral . tDOffset $ tds ! "glyf"
         getGlyph _ 0 = EmptyGlyf
         getGlyph offset _len =

